### ISSUE TITLE
Error after compiler with mingw

### ISSUE THREAD

**Issue by welitonfreitas** [python-pillow/Pillow#2731]
i'm compiler pillow with mingw32 using mysy2 terminal, after compiler and install is show this error on import PIL.Image

```python$ python3 teste.py
Traceback (most recent call last):
  File "teste.py", line 1, in <module>
    import PIL.Image
  File "E:\Pillow\PIL\Image.py", line 56, in <module>
    from . import _imaging as core
ImportError: cannot import name '_imaging'
```
_imaging extension is compiled with name "_imaging-cpython-36m.py", but python not found.

![captura de tela 2017-09-11 as 18 06 54](https://user-images.githubusercontent.com/26512/30296939-5b340768-971c-11e7-9384-5c125b48f488.png)

i'm using python 3.6.2 and mingw gcc 7.2 and openjpeg 2.2.

Att

**Comment by wiredfool** [comment]
We don't have much experience with mingw. However, there are a couple of possible things to check. 

1) Don't test from within the Pillow source directory. There is a PIL directory there without the compiled code, and the current directory is first in the python path. 
2) Try running `python selftest.py --installed` from within the Pillow source directory. That will run a basic sanity and feature test on the installed version, and report the file name of both `Image.py` and the compiled `_imaging` dll/so

**Comment by welitonfreitas** [comment]
I did not have much success...
```python
$ python3 selftest.py --installed
Traceback (most recent call last):
  File "selftest.py", line 11, in <module>
    from PIL import Image, ImageDraw, ImageFilter, ImageMath
  File "C:\msys32\mingw32\lib\python3.6\site-packages\pillow-4.3.0.dev0-py3.6-mingw.egg\PIL\Image.py", line 56, in <module>
ImportError: cannot import name '_imaging'
```
I'll keep investigating..

**Comment by welitonfreitas** [comment]
i'm find this.. and created a #issue in MINGW-packages repositories. 
https://github.com/Alexpux/MINGW-packages/issues/2884

On Makefile ext Suffix for shared library is "-cpython-36m.dll" and correct is ".cpython-36m.dll".
some libraries break down after they are installed

In python3.2 installer from python.org ext Suffix is ".cp36-win32.pyd"

**Comment by wiredfool** [comment]
There should be a fix for this in #2746
